.github/workflows/test.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     strategy:
       matrix:
         php-versions: [7.3, 7.4, 8.0]
-        nextcloud-versions: ['master']
+        nextcloud-versions: ['stable23']
```

Comment:
You could use `${{ github.base_ref }}` :shrug:

Reply:
Seems to break it :(

Reply:
Not in the matrix, if we're only testing against one version, we don't need a matrix